kernel/kernel-defaults/src/test/java/io/delta/kernel/defaults/benchmarks/ReadMetadata.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (2025) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.kernel.defaults.benchmarks;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.nio.file.Path;
+
+/** Workload specification for read_metadata benchmarks. */
```

Comment:
I think you should also explain to the reader what exactly a "read metadata" spec is -- what does it test for? what is an example of it?

Reply:
Updated the docs :) PTAL